pkg/docker/cockpit-atomic-storage

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python2
+#! /usr/bin/python3
```

Comment:
Note that this line (and the similar ones in `pkg/storage`) isn't actually used in production. The script is started via `python.spawn` from JS. So the only real change here is the one to `test-policy`.  Still good to change this line, of course.

Reply:
Right, the commit message also points that out. It's just for cleaning up grepping for old Python 2 stuff.

Reply:
> the commit message also points that out.

Ouch, I should have read that.